client/src/catalog-element-readme.html

```diff
@@ -193,6 +195,17 @@
         '_loadDemos(docsPending)',
       ],
 
+      /**
+       * This handles the hashchange event and finds the element that corresponds based on
+       * Github's rendering of the markdown.
+       */
+      _hashChanged: function() {
+        var hash = window.location.hash.replace('#', '');
```

Comment:
substring(1) because simpler and also the hash is allowed to have its own hash (I do that in iron-doc-viewer!)